Plots/Trajectory/NCL_polyg_14.py

```diff
@@ -0,0 +1,85 @@
+
+###############################################################################
+# Import packages:
+import numpy as np
+import cartopy.crs as ccrs
+import matplotlib.pyplot as plt
+import cartopy.feature as cfeature
+from geographiclib.geodesic import Geodesic
+
+from geocat.viz import util as gvutil
+
+###############################################################################
+# Read in data:
+###############################################
+
+def Plot(color, row, col, pos, title, style, pt):
+
+    fig = plt.figure(figsize=(12,12))
+    ax = fig.add_subplot(row, col, pos, projection=ccrs.PlateCarree())
+    
+    ax.set_extent([-125,-60,15,65], ccrs.PlateCarree())
+    ax.add_feature(cfeature.LAND, color='lightgrey')
+    
+    plt.plot([-120, -64], [20, 60], style, color= color,  transform=ccrs.Geodetic())
+    
+    # start location
+    lon_fr = -120
+    lat_fr = 20
+    # end location
+    lon_to = -64
+    lat_to = 60
+    
+    # This gets geodesic between the two points
+    # WGS84 ellipsoid is used
+    gl = Geodesic.WGS84.InverseLine(lat_fr, lon_fr, lat_to, lon_to)
+    
+    num_points = 10  #for points on geodesic
+    print("distance latitude longitude azimuth")
+    
+    # Compute points on the geodesic, and plot them as red dots
+    # gl.s13 is the total length of the geodesic
+    # the points are equally spaced by true distances, but not on the map
+    # due to the projection distortion
```

Comment:
Can you clarify this? The points are equally spaced using true distances, which makes sense. The second part "but not on the map due to the projection distortion" could use clarification.